config/projects/datadog-agent.rb

```diff
@@ -83,8 +83,10 @@
   # SysVInit service file
   if redhat?
     extra_package_file '/etc/rc.d/init.d/datadog-agent'
+    runtime_dependency 'libcap2'
   else
     extra_package_file '/etc/init.d/datadog-agent'
+    runtime_dependency 'libcap2-bin'
```

Comment:
You shouldn't declare `libcap` as a dependency of the `datadog-agent` package given that you embed the lib with `go-metro`. Tell me if I'm missing something though :)